src/SFA.DAS.Authorization.ProviderPermissions.UnitTests/AuthorizationHandlerTests.cs

```diff
@@ -0,0 +1,160 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ProviderRelationships.Api.Client;
+using SFA.DAS.ProviderRelationships.Types.Dtos;
+using SFA.DAS.ProviderRelationships.Types.Models;
+using SFA.DAS.Testing;
+
+namespace SFA.DAS.Authorization.ProviderPermissions.UnitTests
+{
+    [TestFixture]
+    [Parallelizable]
+    public class AuthorizationHandlerTests : FluentTest<AuthorizationHandlerTestsFixture>
+    {
+        [Test]
+        public Task GetAuthorizationResultAsync_WhenOptionsAreNotAvailable_ThenShouldReturnValidAuthorizationResult()
```

Comment:
Simple rename to GetAuthorizationResult dropping the Async word as the function is now called this

Reply:
Fixed